plugins/spi/SPIOutput.cpp

```diff
@@ -486,6 +502,128 @@ uint8_t SPIOutput::P9813CreateFlag(uint8_t red, uint8_t green, uint8_t blue) {
   return ~flag;
 }
 
+
+void SPIOutput::IndividualAPA102Control(const DmxBuffer &buffer) {
+  // some detailed information on the protocol:
+  // https://cpldcpu.wordpress.com/2014/11/30/understanding-the-apa102-superled/
+  // Data-Struct
+  // StartFrame: 4Byte = 32Bits zeros (APA102_START_FRAME_BYTES)
+  // LEDFrame: 1Byte FF ; 3Byte color info (Blue, Green, Red)
+  // EndFrame: (n/2)bits; n = pixel_count
+
+  // calculate dmx-start-address
+  const unsigned int first_slot = m_start_address - 1;  // 0 offset
+
+  // only do something if minimum 1pixel can be updated..
+  if (buffer.Size() - first_slot < APA102_SLOTS_PER_PIXEL) {
+    OLA_INFO << "Insufficient DMX data, required " << APA102_SLOTS_PER_PIXEL
+             << ", got " << buffer.Size() - first_slot;
+    return;
+  }
```

Comment:
just a style question:
from my feeling i would write this in 'positiv-logic' so i do not have multiple breakouts/returns in the function -
```
// only do something if minimum 1pixel can be updated..
if (buffer.Size() - first_slot >= APA102_SLOTS_PER_PIXEL) {
  // the ungoing function
  .....
} else {
  OLA_INFO << "Insufficient DMX data, required " << APA102_SLOTS_PER_PIXEL
           << ", got " << buffer.Size() - first_slot;
}
```
how do you prefer it?